### DISCUSSION TITLE
How to get browser console log from a log file

### DISCUSSION THREAD

**Discussion by yaoyaoTheCat** [Igalia/cog#470] [General]
Hi,
we have IMX8 and use below arguments when configuring the cog:
```
    "workingDirectory": "/mnt/application/WebApp/bin/",
    "filePath": "chroot",
    "arguments": [
      "--userspec=app:app",
      "/imx",
      "cog",
      "--enable-write-console-messages-to-stdout=true",
      "-b",
      "black"
    ],
    "environment": {
      "XDG_RUNTIME_DIR": "/run/user/1000",
      "HOME": "app",
      "COG_URL": "http://localhost:8080/webapp/index.html",
      "WPE_INIT_VIEW_WIDTH": "1920",
      "WPE_INIT_VIEW_HEIGHT": "1080"
    }
```
 Since this one is configured `"--enable-write-console-messages-to-stdout=true",`  how can we capture the console log in stdout? and how can we capture the stdout console log to a log file by configuring the cog browser?
Thank you

**Comment by philn** [comment]
If cog sends the output to stdout it is up to the calling code (is that some... python code that you shared?) to retrieve it using its language facilities (in python check the subprocess module docs).

**Reply by yaoyaoTheCat** [reply]
Thanks
